sample-apps/02-otel-collector/kubernetes/01-configmap.yaml

```diff
@@ -5,20 +5,55 @@ metadata:
   namespace: otel-collector
 data:
   otel-collector-config.yml: |
+    extensions:
+      sigv4auth:
+        region: "__replace-region__"
+        service: "osis"
+
     receivers:
       otlp:
         protocols:
           grpc:
             endpoint: 0.0.0.0:55680
+          http:
+            endpoint: 0.0.0.0:55681
             
+    processors:
+      memory_limiter:
+        check_interval: 1s
+        limit_percentage: 80
+        spike_limit_percentage: 30
+      batch:
+        send_batch_size: 1000
```

Comment:
For a productive environment, this batch and timeout parameter makes sense, but since during the workshop we generated few data, it will take time to send the events to the OSI.

      batch:
        send_batch_size: 10
        timeout: 10s or even 5s

Reply:
agreed; made the change